pkg/utils/requester_test.go

```diff
@@ -190,12 +190,42 @@ func TestPostObj(t *testing.T) {
 	assert.NotNil(t, err)
 }
 
+// mockLogger exists to check that the logged error is what we expected
+type mockLogger struct {
+	Errors []error
+}
+
+func (t *mockLogger) Debug(message string)   {}
+func (t *mockLogger) Info(message string)    {}
+func (t *mockLogger) Warning(message string) {}
+func (t *mockLogger) Error(message string, err interface{}) {
+	if err, ok := err.(error); ok {
+		t.Errors = append(t.Errors, err)
+	}
+}
+
 func TestGetBad(t *testing.T) {
+	// Using a mockLogger to ensure we're logging the expected error message
+	mlog := &mockLogger{}
+	httpreq := NewHTTPRequester(mlog)
 
-	httpreq := NewHTTPRequester(logging.GetLogger("", ""))
-	_, _, _, err := httpreq.Get("blah12345/good")
-	_, ok := err.(*url.Error)
+	badURL := "blah12345/good"
```

Comment:
NIT: "good" is a little misleading if this is a badURL 🤔

Reply:
added this "htp:/ww.bad-url.fake/blah12345"